src/Libraries/CoreNodeModels/Input/BaseTypes.cs

```diff
@@ -61,6 +95,16 @@ protected override bool UpdateValueCore(UpdateValueParams updateValueParams)
                 Value = value; 
                 return true; // UpdateValueCore handled.
             }
+            if (name == "SerializedWidth" && double.TryParse(value, out double width))
```

Comment:
Use SerializedWidth and SerializedHeight as constant string as it is used multiple times.